data/Remnant Cognizance.txt

```diff
@@ -0,0 +1,626 @@
+	name "Go collect some void sprites"
+	description "Escort the pelican fleet to Nenia and back to Arculus"
+	source
+		government "Remnant"
+	landing
+	destination Viminal
+	waypoint "Nenia"
+	to offer
+		has "Remnant: Cognizance 13: done"
+	on offer
+		conversation
+			`It is almost starting to feel commonplace to have remnant pouncing on your ship and using your cockpit as a meeting room. This time, it is Prefect Chilia and Plume who seem to appear out of nowhere requesting permision to board. The two gather around your console to watch the recording of your trip. They nod at your narration of what transpired, and focus intently when you describe the strange presence in your mind. When you finish recounting your experience, they lean back in thought.`
+			`	"Captain <first>, the exact meaning is unclear, but it seems to be the voice of something not acustomed to speaking to us or creatures like us. Regardless, I think we should move forward with our plan" signs Prefect Chilia. "Best case scenario, we can deprive the Korath of easy access to the void sprites. Worst case scenario, we can move enough that - should the Korath hunt them to the extinction in Nenia, they will continue to exist in our territory."`
+			`	Plume signals his cautious agreement. "This message is intriguing, but I do not think it should be our first priority. That being said, perhaps we should send along some additional remnant who can focus on the possibilities of more such messages."`
+			`	The two look at you, and Chilia signs authoratively, "We are evacuating a large group of void sprites from Nenia to Esquiline. The atmospheres are largely a match, and we have reconfigured the station to provide the nutrients that aren't otherwise present. We have a fleet of Pelicans that have basically been hollowed out so we can lure as many void sprites into it as we can."`
+			choice
+				`	"How are we supposed to lure them inside?"`
+				`	"Are we sure this is a good idea?"`
+					goto goodidea
+				`	"OK, let's go do this."`
+					goto letsgo
+			`	Plume looks a bit embarassed. "Centuries ago when we first studied the void sprites, we learned how to harvest them. As part of that, we discovered that they were particularly interested in certain types of radiation. Likewise, if they discovered a certain kind of element common in some asteroids, they would ignore almost everything while they chewed on it. So, we have emitters inside the cargo holds, along with asteroids to keep them busy for a short while."`
+			choice
+				`	"Are we sure this is a good idea?"`
+				`	"OK, let's go."`
+					goto letsgo
+			label goodidea
+			`	Chilia signs with a resolute tone. "Is it a good idea? Maybe, maybe not. Is it the best idea we have so far? Yes." He sighs "I fully expect the Archon to interfere: they do have long memories. As best we can tell, they can pick out who is leading a fleet, so you should expect to have them talking to you. That is, if they do not simply start shooting, anyway."`
```

Comment:
`	Chilia signs with a resolute tone. "Is it a good idea? Maybe, maybe not. Is it the best idea we have so far? Yes." He sighs, "I fully expect the Archon to interfere: they do have long memories. As best we can tell, they can pick out who is leading a fleet, so you should expect to have them talking to you. That is, if they do not simply start shooting, anyway."`

Reply:
Fixed.